Makefile

```diff
@@ -7,9 +7,9 @@ build_dir=build/pkg
 dist_dir=build/dist
 
 # Variables that are replaced in build time
-bin_path=/usr/bin
-lib_path=/usr/lib/ansible-bundler
-etc_path=/etc/ansible-bundler
+bin_path=.
+lib_path=../lib/ansible-bundler
+etc_path=../../etc/ansible-bundler
```

Comment:
Thank you for your feedback. I understand that changing these variables might prevent the build for other platforms. My intention was to avoid hardcoding the paths, as it could cause issues in different environments. However, I understand that it might not be the best solution for all cases.

Would you have any suggestions on how to make these paths more flexible without affecting other platforms? I am open to any ideas that could help us find a more suitable solution. And please let me know if there are any misconceptions in my understanding.